### ISSUE TITLE
options from CLI should override the options from config file

### ISSUE THREAD

**Issue by Detavern** [golangci/golangci-lint#3291]
### Your feature request related to a problem? Please describe.

Right now, if you use a config file like:

```yml
linters:
  enable:
    - errcheck
    - gosimple
    - govet
    - ineffassign
    - staticcheck
    - typecheck
    - unused
```

And then, you may want disable some plugins for specific projects temporarily.
So you type
```bash
golangci-lint run -D unused
```

then the error comes
```
ERRO Running error: linter "unused" can't be disabled and enabled at one moment
```

Currently, I create a new config file under the project's directory and change it frequently for disable/enable a single linter.
I don't think it is a good practice.

### Describe the solution you'd like.

If we enable/disable some linters from CLI, the same option from config file should be override. 

### Describe alternatives you've considered.

N/A

**Comment by ldez** [comment]
Hello,

This is related how to viper and cobra work.

The CLI configuration already overrides a file configuration but the configuration sources are merged.

So we cannot change this behavior.
